lib/rucio/core/oidc.py

```diff
@@ -18,6 +18,29 @@
 # - Patrick Austin <patrick.austin@stfc.ac.uk>, 2020
 # - Benedikt Ziemons <benedikt.ziemons@cern.ch>, 2021
 # - Radu Carpa <radu.carpa@cern.ch>, 2021
+# - Rob Barnsley <rob.barnsley@skao.int>, 2022
+
+# -*- coding: utf-8 -*-
```

Comment:
Header appears twice 😄

Reply:
no idea how that happened! i've had that before with the add_header script